source/github-helpers/dom-formatters.ts

```diff
@@ -7,7 +7,7 @@ import getTextNodes from '../helpers/get-text-nodes';
 import parseBackticksCore from './parse-backticks';
 
 // Shared class necessary to avoid also shortening the links
-export const linkifiedURLClass = 'rgh-linkified-code';
+export const linkifiedURLClass = '.highlight, .js-suggested-changes-blob';
```

Comment:
If this class isn't shared, it should not be in this file, also because it's not a class anymore but it's a selector